app/src/renderer/vuex/modules/connection.js

```diff
@@ -22,16 +23,16 @@ export default function({ node }) {
 
   const mutations = {
     stopConnecting(state, stop) {
-      state.stopConnecting = stop
+      Vue.set(state, `stopConnecting`, stop)
     },
     setConnected(state, connected) {
-      state.connected = connected
+      Vue.set(state, `connected`, connected)
```

Comment:
This is only necessary if manipulating the property of a property of state.

Reply:
So here I'm abusing the Vue.set function, connected is always a boolean right?

Reply:
I think so. But it's not wrong. ;)